src/manage.c

```diff
@@ -370,7 +367,7 @@ static u8 manage_track_new_acq(void)
   return MANAGE_NO_CHANNELS_FREE;
 }
 
-/** Clear unhealthy flags after some time.  Flags are reset one per day. */
+/** Clear unhealthy flags after some time.  Flags are reset once per day. */
```

Comment:
No, the firmware is clearing the flags once per day (see a couple lines further down).  I can clarify the comment further.